cdp-agentkit-core/cdp_agentkit_core/actions/social/twitter/post_tweet.py

```diff
@@ -0,0 +1,34 @@
+import tweepy
+from pydantic import BaseModel, Field
+
+POST_TWEET_PROMPT = """
+This tool will post text on Twitter."""
+
+class PostTweetInput(BaseModel):
+    """Input argument schema for twitter post text actions."""
+
+    text: str = Field(
```

Comment:
This is confusing that the input is text but the method is `post_tweet`. 

Let's name the input field `tweet` and in the description describe that it is a text string that can be maximum 280 chars.

Reply:
Please reflect these input args and descriptions across the rest of the files / methods